app/services/we_work_remotely.rb

```diff
@@ -0,0 +1,83 @@
+class WeWorkRemotely < JobFetcher
+  BASE_URI = "https://weworkremotely.com/categories/2-programming/jobs.rss"
+
+  DIVS_OR_LIS        = /<\/div>|<li>/
+  NONBREAKING_SPACES = /&nbsp;/
+  NEWLINES_OR_TABS   = /\n+|\t+/
+  ESCAPED_QUOTES     = "\""
+
+  def self.scrape
+    technologies = Technology.pluck(:name)
+    feed = self.pull_feed
+    if feed.entries.present?
+      self.format_entries(feed.entries, technologies)
+    end
+  end
+
+  def self.format_entries(entries, technologies)
+    entries.map do |entry|
+      formatted_entry = self.format_entry(entry, technologies)
+      self.create_records(formatted_entry)
+    end
+  end
+
+  def self.format_entry(entry, technologies)
```

Comment:
We're passing technologies around a lot here - might be better to define it where it's needed or even move the definition of technologies into the JobFetcher class